docs/Reference/Server.md

```diff
@@ -1231,6 +1232,12 @@ fastify.ready(() => {
 })
 ```
 
+### listeningOrigin
+<a id="listeningOrigin"></a>
+
+Immutable instance property indicating the current origin the server
+is listening to, e.g.: `http://127.0.0.1:3000`.
```

Comment:
Maybe append the Unix socket example, otherwise might be a surprise for some people